open-company/open-company-web
# Company Creation w/ placeholder sections

Corresponding API PR https://github.com/open-company/open-company-api/pull/30

To test the dashboard:

1. Use `curl` to create two companies. One with your JWToken, one with a different one.
2. Go to both companies' dashboards and see that placeholder sections are only rendered for the company your token created.
3. Use `curl` to `PUT` a "real section" into the company created by the other token.
4. Go to it's dashboard and see that the real section shows up while placeholder sections are still invisible.

To test rest of the flow:

1. Sign out & go to `/` and notice that companies are shown as usual. **confirmed**
3. Sign in & be redirected to a company you have access to or be redirected to `/create-company` if you don't have access to an existing company
3. Create a company on the `/create-company` page. You will be redirected to the newly created company dashboard.
